http/jetty12/pom.xml

```diff
@@ -181,6 +181,9 @@
                             org.apache.commons.*
                         </Conditional-Package>
                         <Import-Package>
+                            jakarta.annotation.*;resolution:=optional,
```

Comment:
These additions were needed, otherwise the bundle would not start because of missing dependencies. Not sure why though, as the Jetty websocket bundles should be Jakarta agnostic.